client/state/inline-help/test/selectors.js

```diff
@@ -135,3 +138,93 @@ describe( '#getInlineHelpSearchResultsForQuery()', () => {
 		expect( getInlineHelpSearchResultsForQuery( state, 'foo' ) ).to.deep.equal( [] );
 	} );
 } );
+
+describe( 'Admin section search and filters', () => {
+	describe( 'getAdminSectionsResults()', () => {
+		test( 'should return `[]` when there is no search term', () => {
+			const result = getAdminHelpResults( {} );
+			expect( result ).to.deep.equal( [] );
```

Comment:
Do you _need_ to `.deep` here? You just want to assert it's an array no?
```suggestion
			expect( result ).to.equal( [] );
```

Reply:
Test fails if not. Need to research why. Have you run the test with this change?

Reply:
No actually. 

> Test fails if not.

That's a bit odd no?